src/components/Join/Join.tsx

```diff
@@ -0,0 +1,179 @@
+          domain. This gives you more control over your Solid identity.
+        </p>
+        <p>
+          Choose option <i>"Use my existing WebID to access my Pod"</i> with one
+          of these providers
+        </p>
+        <ul>
+          {oidcIssuers
+            .filter(({ server }) => server === 'CSS')
+            .map(({ issuer, registration }) => (
+              <li>
+                <ExternalButtonLink
+                  secondary
+                  href={registration}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {new URL(issuer).hostname}
+                </ExternalButtonLink>
+              </li>
+            ))}
+        </ul>
+        (this will need more explanation),
+        <br />
+        or host your own Solid Server.
+      </>
+    ),
+  },
+]
+
+export const Join = () => {
+  const [modalOpen, setModalOpen] = useState(false)
+  const [selectedTab, setSelectedTab] = useState<string>()
+
+  const handleTabChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setSelectedTab(event.target.value)
+  }
+
+  return (
+    <>
+      <Button primary onClick={() => setModalOpen(true)}>
+        Join
+      </Button>
+      <Modal
+        isOpen={modalOpen}
+        shouldCloseOnEsc
+        shouldCloseOnOverlayClick
+        onRequestClose={() => setModalOpen(false)}
+      >
+        <div className={styles.container}>
+          <div className={styles.content}>
+            To join Sleepy Bike, you need a{' '}
+            <a
+              href="https://solidproject.org/users/get-a-pod"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Solid Pod
+            </a>
+            .
+            <div className={styles.podOptions}>
+              <div className={styles.tabs}>
+                {tabs.map(tab => (
+                  <Fragment key={tab.id}>
+                    <input
+                      type="radio"
+                      id={tab.id}
+                      value={tab.id}
+                      checked={selectedTab === tab.id}
+                      onChange={handleTabChange}
+                    />
+                    <label htmlFor={tab.id}>{tab.label}</label>
+                  </Fragment>
+                ))}
+              </div>
+              <div className={styles.tabContent}>
+                <p>{tabs.find(tab => tab.id === selectedTab)?.content}</p>
+              </div>
+            </div>
+          </div>
+          <div className={styles.footer}>
+            Do you have troubles joining?{' '}
+            <a
+              href="https://matrix.to/#/#ohn:matrix.org"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              Ask for help in our project spaces
+            </a>{' '}
+            or write us email to{' '}
+            <i className={styles.email}>"join at sleepy dot bike"</i>
```

Comment:
We can consider having `support at sleepy dot bike` or similar for more general technical support.

We'll need to add email forwarding in NameCheap and an info who has access and what it is used for in [the wiki](https://github.com/OpenHospitalityNetwork/openHospitality.network/wiki/OHN-instances-and-domains#email-addresses) (which is outdated, btw). Let me know if you wanted me to do it.

Reply:
Changed to `support at sleepy dot bike`. It may make sense to distinguish between general support and tech support, but we can do that later on...